documentation/README.md

```diff
@@ -23,14 +23,43 @@ yarn start
 
 This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
 
-## Including .md file
+## Features And Tools
 
-```console
-{@import <file path>}
-```
+You can find guidance and an introduction to the needed tools and syntax on [our wiki](https://wiki.iota.org/participate/contribute-to-wiki/for_devs/developer_guide)
 
-Example:
 
-```console
-{@import ../../../../bindings/wasm/docs/api-reference.md}
-```
+## Default Structure
+
+IOTA projects usually house their Wiki documentation within the documentation folder of any project. This
+template provides a set structure to separate concerns based on the [Diataxis framework](https://diataxis.fr/), with
+some minor modifications.
+
+You do not need to use all of these sections, so please feel free to delete any which do not suit your needs.
+
+### Getting Started
+
+The [Getting Started folder](documentation/getting_started/_README.md) should be the first section in your documentation. It should give the user a high-level overview of the project, required prior knowledge, prerequisites, and ideally a quick setup guide, or "hello world".
+
+### How
```

Comment:
Shouldn't it be How To or How Tos? Just double checking

Reply:
```suggestion
### How Tos
```